include/boost/geometry/geometries/PolyhedralSurface.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef BOOST_GEOMETRY_GEOMETRIES_POLYHEDRALSURFACE_HPP
+#define BOOST_GEOMETRY_GEOMETRIES_POLYHEDRALSURFACE_HPP
+
+#include <memory>
+#include <vector>
+
+#include <boost/concept/assert.hpp>
+
+#include <boost/geometry/geometries/concepts/point_concept.hpp>
+#include <boost/geometry/geometries/ring.hpp>
+#include <boost/geometry/core/tag.hpp>
+#include <boost/geometry/core/tags.hpp>
+
+#include <boost/config.hpp>
+#ifndef BOOST_NO_CXX11_HDR_INITIALIZER_LIST
+#include <initializer_list>
+#endif
+
+namespace boost { namespace geometry 
+{
+
+namespace model
+{
+
+
+template 
+<
+    typename Point,
+    bool ClockWise = true, bool Closed = true,
+    template<typename, typename> class PointList = std::vector,
+    template<typename, typename> class RingList = std::vector,
+    template<typename> class PointAlloc = std::allocator,
+    template<typename> class RingAlloc = std::allocator
+>
+class PolyhedralSurface : public PointList<ring<Point>, RingAlloc<ring<Point> > >
```

Comment:
I'm quite puzzled by this...
So it inherits from a `vector<ring>` - that sounds reasonable, but why is it called `PointList`? and where is the `RingList` used? But in the type (below) it's defined as a `RingList`.